server/src/lib/db/mysql.ts

```diff
@@ -42,7 +42,17 @@ export default class Mysql {
             DEFAULTS.idleTimeoutMillis,
     };
 
-    this.pool  = mysql.createPool({
+    let isEmptyString = function(str: string): boolean {
```

Comment:
let's move this function into the server utility module (`serser/src/libe/utility.ts`)